configs/osm_config.yaml

```diff
@@ -0,0 +1,401 @@
+osm_clean_columns:
```

Comment:
Please, clould you add a comment before each macroblock of object to describe the object.
That would help readability in the future

Reply:
done.